geode-docs/tools_modules/redis_api_for_geode.html.md.erb

```diff
@@ -105,18 +104,18 @@ start server \
   --J=-Denable-redis-unsupported-commands=true \
   --name=<serverName> \
   --locators=<locatorPort> \
-  --redis-port=<redisPort> \
-  --redis-bind-address=<redisBindAddress> \
-  --redis-password=<redisPassword>
+  --compatible-with-redis-port=<compatibleWithRedisPort> \
+  --compatible-with-redis-bind-address=<compatibleWithRedisBindAddress> \
+  --compatible-with-redis-password=<compatibleWithRedisPassword>
 ```
 
 Commands not listed above are **not implemented**.
 
 ## <a id="advantages-over-redis" class="no-quick-link"></a>Advantages of <%=vars.product_name%> over Redis
 
-<%=vars.product_name%>’s primary advantage is its **scalability**. While the Redis server is single threaded, <%=vars.product_name%> supports high concurrency. Many Redis clients can execute commands on the <%=vars.product_name%> cluster simultaneously.
+<%=vars.product_name%>’s primary advantage is its **scalability**. While the Redis server is single threaded, <%=vars.product_name%> supports high concurrency. Many Redis-compatible clients can execute commands on the <%=vars.product_name%> cluster simultaneously.
 
-<%=vars.product_name%>'s architecture and management features help detect and resolve **network partitioning** problems without explicit management on the part of the Redis client.
+<%=vars.product_name%>'s architecture and management features help detect and resolve **network partitioning** problems without explicit management on the part of the Compatible with Redis Client.
```

Comment:
I think this could also be `Redis client` since we are still talking about actual Redis clients (Jedis, Lettuce, etc.).